articles/cognitive-services/Computer-vision/computer-vision-how-to-install-containers.md

```diff
@@ -105,7 +105,7 @@ Use the [docker run](https://docs.docker.com/engine/reference/commandline/run/)
 * Exposes TCP port 5000 and allocates a pseudo-TTY for the container
 * Automatically removes the container after it exits
 
-```docker
+```DockerFile
```

Comment:
note: this still probably isn't a correct language block because this is actually a block of commands